### ISSUE TITLE
sh in sanhw1

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/CORRECTIONS#341]
@funderburkjim please correct 2 words.

After looking at http://funderburkjim.github.io/cologneapiwork/easyspell/01/ 

```
shiva , slp1=shiva , found=false
shivA , slp1=shivA , found=false
shiba , slp1=shiba , found=false
shibA , slp1=shibA , found=false
shIva , slp1=shIva , found=false
shIvA , slp1=shIvA , found=false
shIba , slp1=shIba , found=false
shIbA , slp1=shIbA , found=false
sHiva , slp1=sHiva , found=false
sHivA , slp1=sHivA , found=false
sHiba , slp1=sHiba , found=false
sHibA , slp1=sHibA , found=false
sHIva , slp1=sHIva , found=false
sHIvA , slp1=sHIvA , found=false
sHIba , slp1=sHIba , found=false
sHIbA , slp1=sHIbA , found=false
ziva , slp1=Siva , found=true
zivA , slp1=SivA , found=true
ziba , slp1=Siba , found=false
zibA , slp1=SibA , found=false
zIva , slp1=SIva , found=false
zIvA , slp1=SIvA , found=false
zIba , slp1=SIba , found=false
zIbA , slp1=SIbA , found=false
```

I wanted to kill all the `sh` variants, as nobody should use them in the book (only as an input option). So I found it it's not true and we have:

```
KshaTrapAvan:IEG
KshAyaTiyaKshAyaTiyAnAm:IEG
KshAyaTiyavazrka:IEG
cash:IEG
cAturmAshyakArikAH:ACC
sannaharsha:MW
shaonanoshao:IEG
```

sannaharsha:MW is a print error, it should have been sannaharSa:MW

cAturmAshyakArikAH:ACC is an OCR error, it should have been cAturmAsyakArikAH:ACC

IEG is dirty all possible ways and sometimes nothing can be done about it. cash:IEG is there an anglicised form of Tamil kāśu.
Khshathrapāvan (IE 8-2, 8-3), Old Persian; ‘a provincial governor.’ See Kṣatrapa. - here it's Old Persian and because cuneiforms are not perfect in transliteration, we can't replace sh with ṣ. So 1 English and rest are Persian words.

**Comment by funderburkjim** [comment]
> sannaharsha MW
Agree it is error.  But the correction is to cerebral sibilant. In slp1 `harza`.  from root 'hfz':
```
cl.1 P. A1. हर्षति, °ते (fr. P. only p. हर्षत्), 
to be excited or impatient, rejoice in the prospect of, be anxious or impatient for (dat.) RV.   ; 
```

Inicidentally, this is a typo (OCR error), not a print error.   In the MW1899 print, 'sh' is the representation
for the SLP1 'z'.  

![image](https://cloud.githubusercontent.com/assets/6393033/24015284/753531a0-0a5e-11e7-85a6-894d4507f1a5.png)

**Comment by funderburkjim** [comment]
confirm acc correction.

**Comment by funderburkjim** [comment]
IEG is, as you say, a special case.  Not clear how to handle. Not worth worrying about now, or in foreseeable future.

**Comment by gasyoun** [comment]
> IEG is, as you say, a special case. Not clear how to handle. Not worth worrying about now, or in foreseeable future.

Right, just keep it out. Persian words are ok, nothing to do with them.

> Inicidentally, this is a typo (OCR error), not a print error. In the MW1899 print, 'sh' is the representation
for the SLP1 'z'.

Jesus Christ.

**Comment by funderburkjim** [comment]
The corrections to MW, ACC now made, and incorporated into sanhw1.
